docs_user/modules/proc_migrating-ovn-data.adoc

```diff
@@ -190,6 +205,8 @@ spec:
       ovnController:
         networkAttachment: tenant
         nodeSelector:
+          node: non-existing-node-name
```

Comment:
why do we need to use such a selector?

Reply:
requested by abokhari but, but afaik not related to this patch , removing